lib/enigma.rb

```diff
@@ -1,8 +1,40 @@
+require 'date'
 class Enigma
+  def rand_key
+    rand(99999)
+  end
```

Comment:
The "rand_key" method will generate a random number to be used as the cipher's key.
It's pretty short and will be in every cipher, should probably make it into an instance variable.